plugins/generator-1.19.4/forge-1.19.4/generator.yaml

```diff
@@ -63,6 +63,7 @@ base_templates:
     name: "@RESROOT/META-INF/accesstransformer.cfg"
     condition_any:
       - ${w.hasElementsOfType('feature')}
+      - ${w.hasElementsOfType('livingentity')}

```

Comment:
Now that #4031 was merged, those checks to enable access transformers (ATs) can probably be fine-tuned only to generate if "workspace has entities with canTrade true". A similar thing can be done with templates for ATs themselves.

@Defeatomizer can help you setup the FTL expression